### PR TITLE
[Batch 5] Porting Notebook PRs

### PR DESCRIPTION
**Pull request by Zsailer**
Porting:
* [x] jupyter/notebook#4328
* [x] jupyter/notebook#4355
* [x] jupyter/notebook#4392
* [x] jupyter/notebook#4412
* [x] jupyter/notebook#4426

### CODE REVIEW

File: jupyter_server/tests/selenium/test_multiselect.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+def test_multiselect(notebook):
```
Comment (kevin-bates) [inline jupyter_server/tests/selenium/test_multiselect.py]: Where are the other selenium tests - in particular the conftest.py file that setups up the 'notebook' instance.

I'm wondering if when the server was setup, the selenium directory was intentionally omitted and this file has "snuck in".

Comment (Zsailer) [reply jupyter_server/tests/selenium/test_multiselect.py]: You're right. I've been trying to catch these selenium tests and delete them. Looks like this one slipped through.

Comment (Zsailer) [reply jupyter_server/tests/selenium/test_multiselect.py]: Removing this directory now.

**Comment by Zsailer** [comment]
Rebased.

**Comment by kevin-bates** [comment]
No cleanup necessary.

**Pull request merged**